lib/web_ui/lib/src/engine/canvaskit/fonts.dart

```diff
@@ -48,23 +48,27 @@ class SkiaFontCollection implements FlutterFontCollection {
   final Map<String, List<SkFont>> familyToFontMap = <String, List<SkFont>>{};
 
   void _registerWithFontProvider() {
-    if (fontProvider != null) {
-      fontProvider!.delete();
-      fontProvider = null;
+    if (_fontProvider != null) {
+      _fontProvider!.delete();
+      _fontProvider = null;
+      skFontCollection?.delete();
+      skFontCollection = null;
     }
-    fontProvider = canvasKit.TypefaceFontProvider.Make();
+    _fontProvider = canvasKit.TypefaceFontProvider.Make();
+    skFontCollection = canvasKit.FontCollection.Make();
+    skFontCollection!.setDefaultFontManager(_fontProvider);
```

Comment:
Call `skFontCollection.enableFontFallback()` here to match how CanvasKit had been setting up the SkParagraph FontCollection

Reply:
Done